src/sentry/integrations/vsts/issues.py

```diff
@@ -38,28 +38,31 @@ def get_project_choices(self, group, **kwargs):
         project_choices = [(project["id"], project["name"]) for project in projects]
 
         params = kwargs.get("params", {})
-        defaults = self.get_project_defaults(group.project_id)
-        try:
-            default_project = params.get(
-                "project", defaults.get("project") or project_choices[0][0]
-            )
-        except IndexError:
-            return None, project_choices
-
-        # If a project has been selected outside of the default list of
-        # projects, stick it onto the front of the list so that it can be
-        # selected.
-        try:
-            next(True for r in project_choices if r[0] == default_project)
-        except StopIteration:
+        if group:
+            defaults = self.get_project_defaults(group.project_id)
             try:
-                project_choices.insert(0, self.create_default_repo_choice(default_project))
-            except (ApiError, ApiUnauthorized):
+                default_project = params.get(
+                    "project", defaults.get("project") or project_choices[0][0]
+                )
+            except IndexError:
                 return None, project_choices
 
+            # If a project has been selected outside of the default list of
+            # projects, stick it onto the front of the list so that it can be
+            # selected.
+            try:
+                next(True for r in project_choices if r[0] == default_project)
+            except StopIteration:
+                try:
+                    project_choices.insert(0, self.create_default_repo_choice(default_project))
```

Comment:
@ceorourke why would our default project be outside the list of choices? Something doesn't seem right here...

Reply:
@scefali 🤷  I didn't write this but I can try to look more into it

Reply:
@ceorourke ah, nvm. Didn't realize this was existing code. Don't worry about that right now.